domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3197,14 +3219,26 @@ public void select(AsyncResult<SQLConnection> conn, String sql, Tuple params,
   }
 
   /**
-   * Run a select query with the readonly connection and return the first record, or null if there is no result.
```

Comment:
`Run an SQL statement that may write data and returns data, for example {@code SELECT nextval('foo')} or {@code UPDATE ... RETURNING ...}`.

Reply:
I resolved this already in the latest commit but Github doesn't recognize the differences.